config/crd/bases/infrastructure.cluster.x-k8s.io_vspheremachines.yaml

```diff
@@ -491,6 +491,10 @@ spec:
                   value in the template from which the virtual machine is cloned.
                 format: int32
                 type: integer
+              os:
```

Comment:
i seem to have a cluster operating with this but i havent added `os` to every CRD (vspheremachinetemplates, vspheremachines, vspherevms).

is OS actually needed in all 3 of these YAMLs ?  or just "vspheremachinetemplates" ? `os:`

Reply:
just the auto-generation at work. CAPV probably needs similar clean up to https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/1506 , but can ignore for now.